src/bindings/python/src/openvino/frontend/onnx/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Package: openvino
+Low level wrappers for the FrontEnd c++ api.
+"""
+
+# flake8: noqa
+
+from openvino.utils import add_openvino_libs_to_path
+
+add_openvino_libs_to_path()
+
+from openvino.frontend.onnx.py_onnx_frontend import ConversionExtensionONNX as ConversionExtension
```

Comment:
what if onnx FE is not available? Will this line throw exception? should we handle it somehow?

Reply:
fixed